app/dao/fact_billing_dao.py

```diff
@@ -255,6 +255,22 @@ def get_rate(
         return 0
 
 
+def get_sms_sent_by_provider_for_current_month():
+    # this query does not include the current day since the task to populate ft_billing runs overnight
+    current_bst_datetime = convert_utc_to_bst(datetime.utcnow())
+    first_day_of_the_month = current_bst_datetime.date().replace(day=1)
+
+    return db.session.query(
+        FactBilling.provider,
+        func.sum(FactBilling.billable_units).label('billable_units_sent')
```

Comment:
This should include rate_multiplier, `func.sum(FactBilling.billable_units * rate_multiplier).label('billable_units_sent')`

Reply:
I've updated this to include rate_multiplier (and the test too to have a row in ft_billing with a different rate_multiplier)